lib/util/http/index.ts

```diff
@@ -57,7 +57,7 @@ function applyDefaultHeaders(options: Options): void {
 // `request`.
 async function gotRoutine<T>(
   url: string,
-  options: GotOptions,
+  options: GotOptions | undefined,
```

Comment:
should never be undefined, so use a default object

Reply:
I added this just for null-check
what is the default object ?